deploy/deploy_contracts.ts

```diff
@@ -159,6 +161,8 @@ const func: DeployFunction = async function(hre: HardhatRuntimeEnvironment) {
             proxy: true,
             args: [Controller.address]
         })
+        // tests expect it to be saved as AdjustableRoundsManager as well
+        await deployments.save("AdjustableRoundsManager", roundsManager)
```

Comment:
This fixes a backward incompatibility of the new `ContractDeployer` with the existing tests. The previous deployer code was also registering the deployed contract with the `contract` (implementation) name. So given some tests look for the `AdjustableRoundsManager` directly, I added the additional `save` here to keep them as is without needing to modify them. I avoided as it would be a larger change, but I can make those changes if you prefer.